container_pipeline/workers/test.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/python
+
+import hashlib
+import json
+import logging
+import os
+
+from container_pipeline.utils import get_job_name
+from container_pipeline.lib.log import load_logger
+from container_pipeline.lib.openshift import Openshift, OpenshiftError
+from container_pipeline.workers.base import BaseWorker
+
+
+class TestWorker(BaseWorker):
+    """Runs the user defined tests on the container built by the build worker."""
+
+    def __init__(self, logger=None, sub=None, pub=None):
+        """Initialize the test worker"""
+        super(TestWorker, self).__init__(logger=logger, sub=sub, pub=pub)
+        self.openshift = Openshift(logger=self.logger)
+
+    def test(self, job):
+        """Runs openshift test job"""
+
+        namespace = get_job_name(job)
+        project = hashlib.sha224(namespace).hexdigest()
+
+        try:
+            self.openshift.login()
+            build_id = self.openshift.build(project, 'test')
```

Comment:
@mohammedzee1000 can you please add a TODO comment pointing [this discussion](https://github.com/CentOS/container-pipeline-service/pull/270/files#r117474305)? I'm requesting this because it's two places we need to address when the issue opened through that discussion (https://github.com/CentOS/container-pipeline-service/issues/276) gets addressed. Something like below should work:

```python
#TODO: This needs to be addressed after addressing Issue #276
build_id = self.openshift.build(project, 'test')
```